operators/src/source/gdal_source/mod.rs

```diff
@@ -427,32 +400,18 @@ where
             },
         );
         let no_data_value = dataset_params.no_data_value.map(T::from_);
-        let fill_value = no_data_value.unwrap_or_else(T::zero);
-        let mut properties = RasterProperties::default();
 
-        tracing::debug!(
-            "no_data_value is {:?} and fill_value is {:?}.",
-            &no_data_value,
-            &fill_value
-        );
+        tracing::debug!("no_data_value is {:?} ", &no_data_value,);
```

Comment:
I guess it suffices to use normal `debug` here.

Reply:
[](https://github.com/geo-engine/geoengine/pull/469/commits/13d5f7f2a6910e94f611ae461997a3156a7fee07)[13d5f7f](https://github.com/geo-engine/geoengine/pull/469/commits/13d5f7f2a6910e94f611ae461997a3156a7fee07)